chromium_src/chrome/browser/ui/views/chrome_typography_provider.cc

```diff
@@ -24,6 +20,10 @@ SkColor ChromeTypographyProvider::GetColor(const views::View& view,
   const ui::NativeTheme* native_theme = view.GetNativeTheme();
   DCHECK(native_theme);
   if (ShouldIgnoreHarmonySpec(*native_theme)) {
+    if (context == views::style::CONTEXT_MENU ||
+        context == views::style::CONTEXT_TOUCH_MENU) {
+      return TypographyProvider::GetColor(view, context, style);
+    }
     return GetHarmonyTextColorForNonStandardNativeTheme(context, style,
                                                         *native_theme);
   }
```

Comment:
Instead of duplicating the code from ChromiumImpl, should we just wrap the section we did change into `if (!ShouldIgnoreHarmonySpec(*native_theme))`?

Reply:
Yes, it also works! Done.